router/router.go

```diff
@@ -289,9 +289,9 @@ type ExpSegment struct {
 	Key string
 }
 
-// ParseExpSegment parses a regexp segment to ExpSegment.
-func ParseExpSegment(exp string) (*ExpSegment, error) {
-	if !strings.HasPrefix(exp, "{") || !strings.HasSuffix(exp, "}") {
+// parseExpSegment parses a regexp segment to ExpSegment.
+func parseExpSegment(exp string) (*ExpSegment, error) {
```

Comment:
Please use a different PR for this

Reply:
Then it is better to leave it to @kdada I think.